src/Events/ViewMatched.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Laravel\Folio\Events;
+
+use Laravel\Folio\MountPath;
+use Laravel\Folio\Pipeline\MatchedView;
+
+class ViewMatched
+{
+    public function __construct(
```

Comment:
Not sure if @taylorotwell is considering this pull request, however, coding style wise, can you add small documentation to this `__construct` like we do every other class of this project, plus remove those `readonly`?

Reply:
Ofcourse, not a problem!